includes/class-wc-gateway-stripe.php

```diff
@@ -506,6 +487,7 @@ public function payment_scripts() {
 		$stripe_params['no_prepaid_card_msg']                     = __( 'Sorry, we\'re not accepting prepaid cards at this time. Your credit card has not been charged. Please try with alternative payment method.', 'woocommerce-gateway-stripe' );
 		$stripe_params['no_sepa_owner_msg']                       = __( 'Please enter your IBAN account name.', 'woocommerce-gateway-stripe' );
 		$stripe_params['no_sepa_iban_msg']                        = __( 'Please enter your IBAN account number.', 'woocommerce-gateway-stripe' );
+		$stripe_params['payment_intent_error']                    = __( 'We could not initiate the payment. Please try again!', 'woocommerce-gateway-stripe' );
```

Comment:
The other error messages are less enthusiastic but less formal, let's be consistent :)
```suggestion
		$stripe_params['payment_intent_error']                    = __( 'We couldn\'t initiate the payment. Please try again.', 'woocommerce-gateway-stripe' );
```